atmo/jobs/urls.py

```diff
@@ -6,6 +6,7 @@
 
 urlpatterns = [
     url(r'^new/', views.new_spark_job, name='jobs-new'),
+    url(r'^taken/', views.taken_spark_job, name='jobs-taken'),
```

Comment:
technically it'd need to be `identifier-taken` which seemed like a weird url :-/